cmd/monaco/main.go

```diff
@@ -149,6 +149,9 @@ func parseInputCommand(args []string, fileReader util.FileReader) (dryRun bool,
 	// Show usage if flags are invalid
 	if environmentsFile == "" {
 		println("Please provide environments yaml with -e/--environments!")
+		println("For deploying a specific project inside a root config folder, run:")
```

Comment:
It would make sense to specify the usage of monaco using the [docopt](http://docopt.org/) CLI description language. For monaco this would be something like:

```
Usage:
   monaco --environments <path-to-environment-yaml-file> [--specific-environment <environment-name>] [--project <project-folder>] [--dry-run] [--verbose] [projects-root-folder]
```

And after that, list all the errors and examples.

Reply:
thanks, i will look into this